packages/store-ui/package.json

```diff
@@ -53,8 +53,8 @@
   },
   "peerDependencies": {
     "@vtex/gatsby-plugin-i18n": "^0.280.0",
-    "react": "^0.0.0-experimental-7f28234f8",
-    "react-dom": "^0.0.0-experimental-7f28234f8"
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
```

Comment:
Do we need to change the version of React? I think the experimental version has some concurrent mode features that we are using.

Reply:
we use this version exact to concurrent mode, but in the theme, on store ui we do not need concurrent mode, and this version broke the storybook runtime.